speech/grpc/src/main/java/com/examples/cloud/speech/StreamingRecognizeClient.java

```diff
@@ -76,6 +81,10 @@ public StreamingRecognizeClient(ManagedChannel channel, String file, int samplin
     this.channel = channel;
 
     speechClient = SpeechGrpc.newStub(channel);
+
+    //Send log4j logs to Console
```

Comment:
Usually, we just configure this with logging.properties, you could comment.  If you are going to run this on GCE, you might wish to integrate w/ gcloud-java's [logging](https://github.com/GoogleCloudPlatform/gcloud-java/blob/master/README.md#stackdriver-logging-alpha).